documentation/guides/migrating-from-v8-to-v9.md

```diff
@@ -100,6 +100,10 @@ A list of functions/mixins and their value equivalents or new token values.
 | `duration(slower)`               | `--p-duration-400`      |
 | `duration(slowest)`              | `--p-duration-500`      |
 
+#### `em()`
+
+Any usage of this function can be hardcoded to `em` units. Alternatively, values can also be hardcoded to `px` units, which will automatically be converted to `rem` in postcss.
```

Comment:
> will automatically be converted to `rem` in postcss.

This is only true if the consumer has a postcss plugin that does this. This repo contains such a plugin but `@shopify/postcss-plugin` doesn't contain anything that does that, any app using sewing-kit would need to opt into using that plugin, which seems like a pretty disruptive migration suggestion.